index.html

```diff
@@ -1288,6 +1286,55 @@ <h2>DID Subject</h2>
 property. The value of <code>id</code> in the resolved <a>DID document</a> is
 expected to match the <a>DID</a> that was resolved.
       </p>
+        </section>
+
+        <section>
+        <h2>alsoKnownAs</h2>
+        <p>
+        A <a>DID controller</a> MAY assert that two or more <a>DIDs</a> identify
+        the same <a>DID subject</a> using the <code>alsoKnownAs</code> property.
+        This property is OPTIONAL.
+        </p>
+
+        <dl>
+        <dt>alsoKnownAs</dt>
+        <dd>
+        The value of <code>alsoKnownAs</code> MUST be a
+        <a data-cite="INFRA#lists">list</a> where each item in the list is a
+        <a>DID</a> that conforms to the rules in Section <a href="#did-syntax"></a>.
```

Comment:
Do we want to allow non-DID URLs as well?

Reply:
It was discussed that we might want `alsoKnownAs` to be restricted to DIDs, and leave `sameAs` for IRIs.... `alsoKnownAs` has the same "type" signature as `owl:sameAs`... I will object to it, based on that being confusing for implementers.

Reply:
Wouldn't that be weird from a (semantic) web perspective, saying that you have to use a different predicate just because the object identifiers are different?

Reply:
yes... it would... thats an argument against using something other than `sameAs` imo.... i remain completely unconvinced that new terms are needed here.... especially when we can add clarity to existing ones....